ansible-collection-microsoft-sql/ansible-collection-microsoft-sql.spec

```diff
@@ -308,71 +140,54 @@ for rolename in %{rolenames}; do
         fi
     fi
 done
-rm %{rolename5}/tests/modules
-rm %{rolename5}/tests/module_utils
-rm %{rolename5}/tests/playbooks/roles
 
 # transform ambiguous #!/usr/bin/env python shebangs to python3 to stop brp-mangle-shebangs complaining
 find -type f -executable -name '*.py' -exec \
      sed -i -r -e '1s@^(#! */usr/bin/env python)(\s|$)@#\13\2@' '{}' +
 
 %build
+# Convert README.md to README.html in the source roles
 readmes=""
-for role in %{rolenames}; do
-    readmes="${readmes} $role/README.md"
+for rolename in %{rolenames}; do
+    readmes="${readmes} $rolename/README.md"
 done
 sh md2html.sh $readmes
 
 mkdir .collections
-%if 0%{?rhel}
-# Convert the upstream collection readme to the downstream one
-%{SOURCE998} lsr_role2collection/collection_readme.md
-%endif
-./galaxy_transform.py "%{collection_namespace}" "%{collection_name}" "%{collection_version}" "Red Hat Enterprise Linux System Roles Ansible Collection" > galaxy.yml.tmp
+# Copy README.md for the collection build
+cp mssql/.collection/README.md lsr_role2collection/collection_readme.md
+# Copy galaxy.yml for the collection build
+cp mssql/.collection/galaxy.yml ./
```

Comment:
Sorry, this comment is about outside of this pr. The galaxy.yml file is for the entire sql collection not just for the mssql | sql-server role. Is it reasonable to have this Description "[The Ansible role that manages Microsoft SQL Server](https://github.com/linux-system-roles/mssql/blob/master/.collection/galaxy.yml#L4)"? I.e., if other roles are added to `sql`, the `README.md` and `galaxy.yml` in `mssql` may need to be updated for the new roles.

Reply:
Here I think it is fine to be fair with users and say straight out what is stored inside the collection. If the collection's readme says "This collection stores content to mange the SQL server" to me it builds expectation that there is more than one role. I want to update it in future if more content is added.

Reply:
![ah](https://user-images.githubusercontent.com/938485/121390792-9319ef00-c902-11eb-9f9d-61d768bab99a.png)
The description `Red Hat Enterprise Linux System Roles Ansible Collection` is displayed like this on the Automation Hub page. This wording was discussed including Terry, Brian and Peter. I suggest to have their inputs, as well. IMHO, the `title` is supposed to be generic and stays the same with the contents grow.

Reply:
Thank's for the details, Noriko. I've updated the description to a generic `Ansible collection for Microsoft SQL Server management`. By updated I mean that I changed it within the galaxy_transform.py script.

Reply:
Thank you, @spetrosi. Your new description looks good to me.